src/Microsoft.OpenApi.Readers/V2/OpenApiOperationDeserializer.cs

```diff
@@ -131,7 +131,12 @@ internal static OpenApiOperation LoadOperation(ParseNode node)
                     operation.RequestBody = CreateFormBody(node.Context, formParameters);
                 }
             }
-            
+
+            foreach (var response in operation.Responses.Values)
+            {
+                ProcessProduces(response, node.Context);
```

Comment:
>    ProcessProduces(response, node.Context); [](start = 12, length = 44)

Isn't this handled in OpenApiResponseDeserializer.cs line 123?

Reply:
I had to put it in operation for the case when produces came after responses.  But I needed it also in responses for when responses are defined at the root level as a reusable thing.  So I made it work when you call it in both places.

Reply:
Tricky! Sounds good to me.

---
In reply to: [193229384](https://github.com/Microsoft/OpenAPI.NET/pull/267#discussion_r193229384) [](ancestors = 193229384)